understanding/20/contrast-minimum.html

```diff
@@ -13,15 +13,14 @@ <h1>Understanding Contrast (Minimum)</h1>
       <h2>Intent of Contrast (Minimum)</h2>
       
       
-	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with moderately low vision.
+	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with limited vision, including color vision deficiencies.
     This user group does not typically use contrast-enhancing assistive technology. 
     For people without color deficiencies, hue and saturation have minimal or no effect on legibility as assessed by reading performance (Knoblauch et al., 1991).
-    Color deficiencies can affect luminance contrast somewhat. 
-    Therefore the contrast is calculated in such a way that color (hue) is not a key factor. 
-    People who have a color vision deficit (often referred to as "color blindness") will also have adequate contrast between the text and the background.</p> 
+    Some vision impairments impact perception of luminance contrast. 
+    People who have a color vision deficit (often colloquially referred to as <q><olor blindness</q>) will also require adequate contrast between the text and the background.</p> 
```

Comment:
an open bracket had been entered where there should be a "c" in the quote color blindness
```suggestion
    People who have a color vision deficit (often colloquially referred to as <q>color blindness</q>) will also require adequate contrast between the text and the background.</p> 
```